packages/default-theme/src/components/forms/SwAddressManager.vue

```diff
@@ -39,7 +39,7 @@
         class="title desktop-only"
         :level="4"
       />
-      <div v-if="isEditModeOpen" class="address-manager__list">
+      <div v-if="!isEditModeOpen" class="address-manager__list">
```

Comment:
this code is from another PR, shouldn't be here

Reply:
I accidentally checkout from another branch, I will revert it tomorrow.
To the new component, I've just thought if anyone want to use top navigation component, they could use it for avoiding crash.
Any advices here?

Reply:
hmm, the problem is more related to some potential bug in Storefront UI component `SfHeader`. Maybe it's not happening anymore after recent library updates?
Did you try to do reproduction steps from the issue? Could you describe the results?